tests/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.9-slim as builder
+
+WORKDIR /usr/src/app
+
+RUN pip install poetry
+
+COPY pyproject.toml poetry.lock ./
+
+RUN poetry export -f requirements.txt > requirements.txt
```

Comment:
No, README.md is created for test project description, how to use, etc, and will be updated later.
Requirements.txt is not needed because package management is done using Poetry (pyproject.toml, poetry.lock).
Dockerfile uses poetry for generating requirements.txt during docker build.